concepts/index.md

```diff
@@ -3,47 +3,56 @@ layout: single
+### Cluster
 
-## Projects
+You can define *Clusters*, which are logical groupings of Server Groups in Spinnaker.
 
-Projects are arbitrary collections of infrastructure resources and pipelines. This is a convenience feature which allows you to create a dashboard containing Applications, Clusters and Pipelines by wildcard.
+### Load Balancer
 
-<!-- ## Infrastructure
+A *Load Balancer* is associated with an ingress protocol and port range. It balances traffic among instances in its Server Group. Optionally, you can enable health checks for a load balancer, with flexiblity to define health criteria and specify the health check endpoint.
 
-This is a catch net area from which potentially lists all infrastructure components across all of Spinnaker that you have access to. You can search for Applications, Clusters, Server Groups, Instances, Load Balancers and Security Groups by wildcard. -->
+### Security Group
 
-## Load Balancers
+A *Security Group* defines network traffic access. It is effectively a set of firewall rules defined by an IP range (CIDR) along with a communication protocol (e.g., TCP) and port range.
 
-Load Balancers route incoming traffic to associated and enabled Server Groups.
+> Learn more on the [Clusters](/concepts/clusters/) page.
 
-## Security Groups
+## Deployment Management
 
-Security Groups define traffic ingress rules for Server Groups.
+You use Spinnaker's deployment management features to construct and manage continuous delivery workflows. 
 
-## Resource Mappings from Spinnaker to Provider
+### Pipeline
 
-Spinnaker | AWS | GCE | Kubernetes | Azure | OpenStack | Cloud Foundry
----|---|---|---|---|---|---
-Server Group | ASG | MIG | Replica Set
-Load Balancer | Load Balancer | L3/L4/HTTP/internal Load Balancer | Service
-Security Group | Security Group | Firewall Rule | Ingress Rule
+![](pipelines.png)
+
+*Pipelines* are the key deployment management construct in Spinnaker. They consist of a sequence of actions, known as stages. You can pass parameters from stage to stage along the pipeline. You can start a pipeline manually, or you can configure it to be started by automatic triggering events, such as a Jenkins job completing, a new Docker image appearing in your registry, a CRON schedule, or a stage in another pipeline. You can configure the pipeline to emit notifications to interested parties at various points during pipeline execution (such as on pipeline start/complete/fail), by email, SMS or HipChat.
+
+### Stage
+
+A *Stage* in Spinnaker is an action that forms an atomic building block for a pipeline. You can sequence stages in a Pipeline in any order, though some stage sequences may be more common than others. Spinnaker comes pre-packaged with a number of stages such as Deploy, Resize, Disable, Manual Judgment, and many more. You can see the full list of stages and read about implementation details for each provider in the [Reference](/reference/providers) section.
```

Comment:
"pre-packaged" doesn't feel right here. It implies that the user can package other stages of their own, which they can't. If it was "pre-packaged _pipelines_" that would make more sense, because a user _can_ create their own pipelines.

"Spinnaker stages include Deploy, Resize... and many more"